geometric_features/aggregation/__init__.py

```diff
@@ -43,7 +43,7 @@ def get_aggregator_by_name(region_group):
                                           'date': '20201130',
                                           'function': arctic_seaice},
                'Ocean Basins': {'prefix': 'oceanBasins',
-                                'date': '20200621',
+                                'date': '20240830',
```

Comment:
@cbegeman, I updated the date stamp for Ocean Basins.  I hope you don't mind.

This is needed so new region masks get created in MPAS-Analysis and Compass that include Southern Hemisphere.